### ISSUE TITLE
Add conditional launch of expirationd tasks

### ISSUE THREAD

**Issue by no1seman** [tarantool/expirationd#116]
Often, when cluster is bootstrapped from the scratch there is no any user spaces at all and developer needs to implement custom logic to start expirationd task only after space(s) to be proceesing by task will be created.

It wiil be great if this kind of machinery will be already onboard.

**Comment by Totktonada** [comment]
NB: I have added it to https://github.com/tarantool/expirationd/issues/68#issuecomment-1148585261, but I don't mind tracking it as a separate task.

**Comment by oleg-jukovec** [comment]
The issue may also be related to the bug #64.
